diff --git a/config.def.h b/config.def.h
index 1edb647..0dbae10 100644
--- a/config.def.h
+++ b/config.def.h
@@ -4,14 +4,16 @@
 static int topbar = 1;                      /* -b  option; if 0, dmenu appears at bottom     */
 /* -fn option overrides fonts[0]; default X11 font or font set */
 static const char *fonts[] = {
-	"monospace:size=10"
+	"JetBrains Mono Medium:size=10:antialias=true:autohint=true",
+  "Hack Nerd Font:pixelsize=10:antialias=true:autohint=true", // Powerline
+  "Symbola:pixelsize=10:antialias=true:autohint=true" // Unicode
 };
 static const char *prompt      = NULL;      /* -p  option; prompt to the left of input field */
 static const char *colors[SchemeLast][2] = {
 	/*     fg         bg       */
-	[SchemeNorm] = { "#bbbbbb", "#222222" },
-	[SchemeSel] = { "#eeeeee", "#005577" },
-	[SchemeOut] = { "#000000", "#00ffff" },
+	[SchemeNorm] = { "#d8dee9", "#1d1f21" },
+	[SchemeSel] = { "#268bd2", "#444" },
+	[SchemeOut] = { "#87cdff", "#444" },
 };
 /* -l option; if nonzero, dmenu uses vertical list with given number of lines */
 static unsigned int lines      = 0;
